src/javascripts/button_state.js

```diff
@@ -0,0 +1,35 @@
+// Adds 'loading icon' to button with id of 'btn__loading' when clicked
+
+(function () {
+	var btnLoad, input;
+
+	// Grab elements with id of btn__loading
+	btnLoad = document.getElementById('btn_loading');
```

Comment:
What if there's more than one button with the ID of btn_loading (on the same page)? HTML spec says IDs should be unique.